docs/installation.rst

```diff
@@ -86,11 +93,15 @@ Some older operating systems may require you to specify Python version 3, like t
         * `win64_mingw53`, `win32_mingw53`,
         * `win64_msvc2019_winrt_x64`, `win64_msvc2019_winrt_x86`, `win64_msvc2019_winrt_armv7`
         * `win64_msvc2017_winrt_x64`, `win64_msvc2017_winrt_x86`, `win64_msvc2017_winrt_armv7`
-    * For android and Qt 5.13 or below, choose one of: `android_x86_64`, `android_arm64_v8a`, `android_x86`,
-      `android_armv7`
-* You can specify external 7zip command path instead of built-in extractor.
-* When specifying `all` for extra modules option `-m` all extra modules are installed.
+    * For android and Qt 5.11 or below, choose one of: `android_x86`, `android_armv7`
+    * For android and Qt 5.12 or Qt 5.13 or Qt 6, choose one of:
+      `android_x86_64`, `android_arm64_v8a`, `android_x86`, `android_armv7`
+* You can specify external 7zip command path instead of built-in extractor by using the ``-E`` or ``--external`` flag.
```

Comment:
I don't think that the `--external` option is important enough to tell people about it in the tutorial, so I'm going to leave it out. Please let me know if you disagree.

Reply:
Yes, some users need to use the option. It can be in the tutorial.